Android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu May 24 11:12:39 IST 2018
+#Tue Jul 03 11:13:27 CEST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.5-all.zip
```

Comment:
It's still here.

Reply:
isit? click on the file and look at it raw. its not there and the line is red with a (-) or do you mean both should be removed?

Reply:
I am just saying that this change should not be here. 
So, you should remove this file from the commit